Source/NSCollectionView.m

```diff
@@ -664,6 +979,9 @@ - (void) encodeWithCoder: (NSCoder *)aCoder
       //[aCoder encodeCGFloat: _verticalMargin forKey: NSCollectionViewVerticalMarginKey];
       [aCoder encodeObject: _backgroundColors 
                     forKey: NSCollectionViewBackgroundColorsKey];
+
```

Comment:
Looks like you are not encoding most of the new values you decode.

Reply:
I should be encoding all of them.   The only new ones are the BackgroundColors and collectionViewLayout keys.

Reply:
Could you please clarify which ones you believe are not being encoded?  It looks to me like the same keys are being decoded in initWithCoder: as those encoded in encodeWithCoder: @fredkiefer

Reply:
Same with above.